### PR TITLE
Support firefox browser

### LINKED ISSUE #5: Need Firefox extension :)
This is great!

Would it be possible to have a Firefox extension as well or maybe to make it cross-browsers (http://userscripts.org/)?

### PR DESCRIPTION
**Pull request by asdofindia**
This must fix #5 (for firefox nightlies at least, now)

Since [storage API is not supported in content scripts](https://developer.mozilla.org/Add-ons/WebExtensions/Chrome_incompatibilities#storage) in firefox now, what I've done is check if chrome.storage is available and if not, automatically assume the user needs the cubes and make the cubes active.

I've added run_at in manifest, and set it to document_start because the [default in firefox is document_idle](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/content_scripts)

Also, I've set the extension id to be isometric-contributions@jasonlang.me which I hope is fine. (It's not displayed anywhere I think)

Do let me know if there's anything else required for this to be merged.

### CODE REVIEW

**Comment by jasonlong** [comment]
Thanks @asdofindia. Would you mind updating the README with instructions on how to install the Firefox extension?

**Comment by asdofindia** [comment]
@jasonlong yes, I can add instructions. But need some info.

Should I add an xpi file and save people the trouble of [packaging it](https://developer.mozilla.org/Add-ons/WebExtensions/Packaging_and_installation) themselves? 

If yes, how to organize the folder?

**Comment by jasonlong** [comment]
I'm thinking it would be easiest for users if there was a separate top level `firefox` directory. I already have that pattern in place for Chrome and Safari. It's not ideal that there's a good bit of duplication, but I'm ok with it as long as there aren't a ton of browsers.

If we go that route, I'm thinking we revert the changes here and apply them only to the Firefox directory. I'm not very familiar with Firefox extensions, but the xpi file sounds like a good idea.

**Comment by asdofindia** [comment]
I've put everything in a folder inside firefox folder and then put the xpi file inside the firefox folder. I've also added some instructions in readme. (Unfortunately, addons.mozilla.org is not accepting web extensions at the moment and therefore we can't submit it there yet) 

If you're even slightly less than happy, let me know what to fix :)

File: firefox/isometric-contributions/manifest.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "storage"
+  ],
+  "icons" : {
+    "48" : "icon-48.png",
+    "128" : "icon-128.png"
+  },
+  "manifest_version": 2,
+  "applications": {
+      "gecko": {
+          "id": "isometric-contributions@jasonlang.me"
```
Comment (jasonlong) [inline firefox/isometric-contributions/manifest.json]: Small typo - should be jasonlong.me 😄

**Comment by jasonlong** [comment]
Other than that one typo, this looks great @asdofindia. Thanks for putting this together!

**Comment by asdofindia** [comment]
Ah, sorry for that. :P Updated the branch.

**Pull request merged**